contestreminder/templates/user/profile.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}Profile{% endblock %}
+
+{% block content %}
+	<div class="bg-text">
+		<img class="img" src="{% static 'images/register.gif' %}" height="150px" alt="Register">
+		<h1 style="color:black;">Profile</h1>
+		<form method="post">
+			{% csrf_token %}
+			
+			{% for error in form.non_field_errors %}
+			    <span style="color: red; font-size: 12px;">{{error}}</span>
+			{% endfor %}
+
+			</br>
+			
+			{% for field in form.visible_fields %}
+		    	{{ field.label_tag }}
+		    	<span class="imp">{{ field }}</span></br></br>
+				{{ field.errors }}
+			{% endfor %}
+
+			<a class="btn2" href="{% url 'home' %}">Home</a> </br></br>
+
+			<button class="btn1" type="submit">Update</button> &emsp;
+			<a class="btn1" href="{% url 'delete' %}">Delete Account</a>
```

Comment:
1) Please, place the home button below the Update and delete the account buttons.
Also, the Home and Delete update button should be after </form> tag.
2) give btn2 class to Delete account.

Reply:
"Also, the Home and Delete update button should be after tag."

I don't think I quite understand this part. Could you rephrase it, please?

Reply:
after </form> tag.
Sorry for the typo.